### PR TITLE
add ReturnArrowWhitespaceRule

### PR DESCRIPTION
**Pull request by akirahrkw**
Hi @jpsim and everyone
I've added a rule which checks whether you have 1 space before return arrow and return type or not
ex:
 Good: func abc() -> Int {}
 Bad:  func abc()->Int {}

But I'm not sure whether this rule makes sense for you or not. 
and I think this rule is not on github's swift style guide though, we have our own swift style guide for our dev team, https://github.com/burpple/swift-style-guide (which is not completed yet:) )
so that would be nice if I can contribute and apply some of our rules to this project.

thanks:)

### CODE REVIEW

File: Source/SwiftLintFrameworkTests/StringRuleTests.swift [hunk 1]
```diff
@@ -70,6 +70,12 @@ class StringRuleTests: XCTestCase {
             commentDoesntViolate: false)
     }
 
+    func testLinesShouldContainReturnArrowWhitespace() {
+        verifyRule(ReturnArrowWhitespaceRule().example,
+            type: .ReturnArrowWhitespace,
+            commentDoesntViolate: false)
```
Comment (jpsim) [inline Source/SwiftLintFrameworkTests/StringRuleTests.swift]: I think this should be true (or just omitted)

File: Source/SwiftLintFramework/Rules/ReturnArrowWhitespaceRule.swift [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        triggeringExamples: [
+            "func abc()->Int {}\n",
+            "func abc()->[Int] {}\n",
+            "func abc()-> Int {}\n",
+            "func abc() ->Int {}\n",
+            "func abc()  ->  Int {}\n",
+            "var abc = {(param: Int) ->Bool in }\n",
+            "var abc = {(param: Int)->Bool in }\n"
+        ],
+        showExamples: false
```
Comment (jpsim) [inline Source/SwiftLintFramework/Rules/ReturnArrowWhitespaceRule.swift]: Why not show these examples?

**Comment by jpsim** [comment]
This looks great, thanks! I just have very minor feedback. It'd be nice if you could add an entry for this in the CHANGELOG too!

**Comment by akirahrkw** [comment]
@jpsim thanks for your feedback!
I will push the change soon, but should I add an entry on 0.1.1 part in CHANGELOG? or better to make 0.1.2?

**Comment by jpsim** [comment]
Ah, I just created an empty "master" section to the changelog, please add your entry there.

**Comment by akirahrkw** [comment]
thanks:) i just pushed my commits!

**Comment by jpsim** [comment]
Great work! Thanks!

**Pull request merged**